native/src/main/resources/kafka_logging.properties

```diff
@@ -1,8 +0,0 @@
-handlers = java.util.logging.ConsoleHandler
-java.util.logging.ConsoleHandler.level = ALL
-java.util.logging.ConsoleHandler.formatter = java.util.logging.SimpleFormatter
-java.util.logging.SimpleFormatter.format = [%1$tc] %4$s: %2$s - %5$s %6$s%n
```

Comment:
Dont we need this format to be set from the `initializeLoggingConfigurations` function?

Reply:
Updated in https://github.com/ballerina-platform/module-ballerinax-kafka/pull/901/commits/89d1eda496684044f23165793fa3c2c227e734d6